ifSwitchTernary/task4.js

```diff
@@ -0,0 +1,36 @@
+// 3.Преобразовать Task 2 - 1 таким образом, чтобы значение НАПРИМЕР '2' (т.е. ЛЮБАЯ строка в которой лежат ТОЛЬКО ЦИФРЫ) пропускалось,
+// преобразовываясь в number
+const age = prompt("What is your age?");
+const minAge = 18;
+const maxAge = 60;
+
+const checkNumIsString = 0 + (+ age);
+
+if (typeof checkNumIsString != "number") {
```

Comment:
typeof NaN === 'number'
```suggestion
if (!isNaN(checkNumIsString)) {
```